osu.Game/Scoring/Legacy/LegacyScoreDecoder.cs

```diff
@@ -65,6 +65,10 @@ public Score Parse(Stream stream)
 
                 scoreInfo.Mods = currentRuleset.ConvertFromLegacyMods((LegacyMods)sr.ReadInt32()).ToArray();
 
+                // lazer replays get a really high version number.
+                if (version < 30000000)
```

Comment:
```suggestion
                if (version < LegacyScoreEncoder.LATSET_VERSION)
```
not sure whether the comment above it should stay or not though.

Reply:
it can't use the constant as it is intended to be changeable if we ever need to increment it (which is why it's called "LATEST").